src/IB/IBFEMethod.cpp

```diff
@@ -675,7 +673,7 @@ IBFEMethod::interpolateVelocity(const int u_data_idx,
         for (int ln = 0; ln <= getFinestPatchLevelNumber(); ++ln)
         {
             d_secondary_hierarchy
-                .getPrimaryToScratchSchedule(ln, u_data_idx, u_data_idx, d_ib_solver->getVelocityPhysBdryOp())
+                ->getPrimaryToScratchSchedule(ln, u_data_idx, u_data_idx, d_ib_solver->getVelocityPhysBdryOp())
```

Comment:
I'm surprised that this is clang-format approved. 😀

Reply:
I think you're right - it looks like I didn't run it after all. I need to spend some real time getting the CI set up at some point.

Reply:
That change is OK, but others were not 😆